humanguide/design-concepts/units.rst

```diff
@@ -0,0 +1,61 @@
+Units
+=====
+
+Resolution Independence Approach
+--------------------------------
+
+The objective of resolution independence is to make it easy for graphical user interfaces in Ubuntu Touch to scale to all the form factors that Ubuntu Touch targets: phones, tablets, laptops and desktops. The approach taken combines simplicity for the designers and developers with visual fidelity, quality and usability.
+
+Measurement Units
+-----------------
+
+Grid Unit
+~~~~~~~~~
+
+A measurement unit called the *grid unit* is defined, abbreviated gu. 1 grid unit translates to a given number of pixels depending on the type of screen that the user interface is displayed on. For example, on a laptop computer 1 grid unit will typically translate to 8 pixels. The number of pixels per grid unit is chosen in order to preserve the perceived visual size of UI elements and therefore depends on the density of the display and the distance the user is to the screen. We also ensure that 1 grid unit is always an integer number of pixel.
```

Comment:
You may wish to italicize "gu" as well here, else the end of the sentence is difficult to discern. Periods can be used to denote abbreviations in English.